### PR TITLE
Rephrase wrongly understood rule about playing attack/defense

### PR DESCRIPTION
**Pull request by DarthGandalf**
en-both-32

### CODE REVIEW

File: sections/combat.tex [hunk 1]
```diff
@@ -82,7 +82,7 @@ \subsection*{\pagetarget{CombatCards}{Using Cards During Combat}}
 Ongoing \includesvg[height=10px]{\svgs/ongoing.svg} and \includesvg[height=10px]{\svgs/activation.svg} Activate effects can be used only \textbf{when Activating one of your Units and before it attacks}.
 Ongoing effects\index{Ongoing Effects} last until end of Combat or if the effect on the card is used up.\par
 Instant \includesvg[height=10px]{\svgs/instant.svg} Cards may be played \textbf{at any time} except between rolling the Combat Die and resolving damage unless otherwise stated.
-Instant Cards which increase a Unit's \includesvg[height=10px]{\svgs/attack.svg} or \includesvg[height=10px]{\svgs/defense.svg}, such as the Attack and Defense Statistic cards, \textbf{must} be played after an attack has been announced and persist only for the duration of that attack.
+Effects of Instant Cards, such as the Attack and Defense Statistic cards, which increase a Unit's \includesvg[height=10px]{\svgs/attack.svg} or \includesvg[height=10px]{\svgs/defense.svg}, expire whenever an attack happens or the Activation ends, whichever comes first.
```
Comment (tomaas-zeman) [inline sections/combat.tex:85]: Does "attack happens" include even the retaliation phase?

Imagine this edge case.
1. I decide to attack and use +1 defense card
2. I roll the attack die and hit the enemy creature
3. Enemy creature now retaliates
4. Do I still have +1 defense on my creature for the retaliation attack or not?

I know it's a bit weird to use defense card before my attack but that would make me better understand the time frame of card effects

Comment (qwrtln) [reply sections/combat.tex:85]: My guess would be no, you don't have +1 defense.

Comment (tomaas-zeman) [reply sections/combat.tex:85]: I would think so too.

I would maybe use word "resolves" instead of "happens". Otherwise the change sounds good to me

Comment (DarthGandalf) [reply sections/combat.tex:85]: Good question.

Выделение_577

I need to think how to phrase this.

Comment (DarthGandalf) [reply sections/combat.tex]: Done. PTAL

Comment (qwrtln) [reply sections/combat.tex]: LGTM. Nothing left to guesses now.

**Review by qwrtln — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**